viz_scripts/scaffolding.py

```diff
@@ -54,9 +54,8 @@ def get_participant_uuids(program, load_test_users):
     disp.display(participant_list.user_email)
     return participant_uuid_str
 
-async def add_base_mode_footprint(trip_list):
+async def add_base_mode_footprint(trip_list, labels):
```

Comment:
Is there a reason this one is still `async`?

I think if you make this one synchronous, you can also make `load_all_confirmed_trips`, `load_all_participant_trips`, and the `load_viz_notebook_*` functions all synchronous.

Reply:
Yes, we are calling `await emffc.calc_footprint_for_trip()` for the calculation of footprint for the trip. Therefore this function needs to be an asynchronous one. That in turn has resulted in remaining `load_all_confirmed_trips`, `load_all_participant_trips` and `load_viz_notebook_*` functions to remain async.

Reply:
Ah, I see now. Thanks for explaning